id.sch.smkn2kediri.UNBK.appdata.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- See https://wiki.gnome.org/GnomeGoals/AppDataGnomeSoftware -->
+<component type="desktop">
+  <id>id.sch.smkn2kediri.UNBK.desktop</id>
+  <metadata_licence>ISC</metadata_licence>
+  <name>UBNK</name>
+  <summary>Indonesian Exam Browser</summary>
+  <description>
+    <p>
+      Indonesian Exam Browser
```

Comment:
We're going to need a better description from the Indonesia team, but we can do that in a follow-up task.